web-ng/src/app/components/feature-panel-header/feature-panel-header.component.ts

```diff
@@ -130,6 +134,34 @@ export class FeaturePanelHeaderComponent
     });
   }
 
+  getFeatureName(): string | number {
```

Comment:
@DaoyuT This is the method we were talking about that may need to be refactored (and `| number` removed`).

Reply:
`FeatureService` could be a good candidate.